tensorflow/python/autograph/impl/conversion.py

```diff
@@ -496,7 +496,7 @@ def convert_entity_to_ast(o, program_ctx):
             keyed by their symbol name.
 
   Raises:
-    ValueError: if the entity type is not supported.
+    NotImplementedError: if the object conversion is not yet supported.
```

Comment:
To be more consistent with the docstring, consider something like: "if entity is of a type that is not yet supported".